go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2371,6 +2371,26 @@ func (s *VtctldServer) RunHealthCheck(ctx context.Context, req *vtctldatapb.RunH
 	return &vtctldatapb.RunHealthCheckResponse{}, nil
 }
 
+func (s *VtctldServer) GetPermissions(ctx context.Context, req *vtctldatapb.GetPermissionsRequest) (*vtctldatapb.GetPermissionsResponse, error) {
+	span, ctx := trace.NewSpan(ctx, "VtctldServer.GetPermissions")
+	defer span.Finish()
+
+	span.Annotate("tablet_alias", topoproto.TabletAliasString(req.TabletAlias))
+	ti, err := s.ts.GetTablet(ctx, req.TabletAlias)
+	if err != nil {
+		return nil, err
```

Comment:
i would recommend wrapping this particular case in `vterrors.Errorf(vtrpc.Code_NOT_FOUND, ...)` (there are other examples here you can refer to!)

Reply:
shouldn't we making it consistent across other apis as well .. what do you recommend if I can change other in same PR or create another PR for changing across other API as well.. e.g healthcheck etc.

Reply:
Different failures have different semantics. Let's just focus on this one in this PR

Reply:
Let just do it in another PR ..

Reply:
Well, I think we should introduce the pattern we want _here_, now, rather than merge something we know we immediately want to change. I'm just saying we should do it everywhere _else_ right now.